e2e/portal/tests/ProfilePageActivityLog/ViewActivityOverview.spec.ts

```diff
@@ -167,13 +170,75 @@ test.describe('as admin user', () => {
   });
 });
 
-test.describe('as user with fewer permissions', () => {
-  test.beforeEach(async ({ page }) => {
+test.describe('as user with only view paper voucher permissions', () => {
+  test('transaction row contains current balance, but not "view voucher" button', async ({
+    page,
+  }) => {
+    // Login as admin
+    await loginAs({
+      page,
+      username: env.USERCONFIG_121_SERVICE_EMAIL_ADMIN,
+    });
```

Comment:
Is this separate login needed? Maybe you can just add the permissions you need to the user you are creating down below? So that it can 'create payment' etc, but also still keep the relevant difference between FSP-type voucher read permissions.

Or alternatively: make the whole setup shared across all tests in this file, putting it in the general beforeEach. Then again, this setup is just for this test, which pollutes the other test. So maybe not actually.